DQMOffline/Trigger/plugins/HTDQM.cc

```diff
@@ -0,0 +1,83 @@
+#include "DQMOffline/Trigger/plugins/HTDQM.h"
+
+HTDQM::HTDQM(){}
+
+HTDQM::~HTDQM(){}
+
+void HTDQM::initialise(const edm::ParameterSet& iConfig ){
+
+  ht_variable_binning_ = iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<std::vector<double> >("htBinning");
+  met_variable_binning_ = iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<std::vector<double> >("metBinning2");
+  ht_binning_ = getHistoPSet(iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("htPSet"));
+  ls_binning_ = getHistoLSPSet (iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("htlsPSet"));
+  
+  htME_variableBinning_.numerator   = nullptr;
```

Comment:
hi @tankit - these initializations are not needed (they are already done for you)